webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -211,7 +218,12 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
                         name={['bindings', name, 'autoCreateNamespace']}
                         valuePropName="checked">
                         <Toggle
-                          onChange={() => form.setFieldValue(['resourceQuotas', name], undefined)}
+                          onChange={() => {
+                            form.setFieldValue(['resourceQuotas', name], undefined);
+                            if (loadedResourceQuotaList[name]) {
+                              delete loadedResourceQuotaList[name];
+                            }
```

Comment:
Could you talk a bit about what the intent is doing this here + in the submit? React can't pick up changes that happen out of state, so this introduces state that's out of sync with the render, which we want to avoid if at all possible.

Reply:
talked to John and decided to use a different approach